_posts/en/newsletters/2019-08-07-newsletter.md

```diff
@@ -0,0 +1,148 @@
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the maintenance release of Bitcoin Core
+0.18.1, summarizes a discussion about BIP174 extensions, and notes a
+proposal for LN trampoline payments.  Our *bech32 sending support*
+section this week features a special case study contributed by BRD and
+our *notable changes* section highlights several possibly significant
+developments.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to Bitcoin Core 0.18.1:** this maintenance release makes
+  available several bug fixes and other improvements.  [Upgrading][bitcoin
+  core 0.18.1] is recommended.
+
+## News
+
+- **BIP174 extensibility:** the author of the Partially-Signed Bitcoin
+  Transactions (PSBTs) specification, Andrew Chow, [proposed][psbt
+  extensions] a few minor changes for general adoption:
+
+    - *Reserved types for proprietary use:* some applications are
+      already including data in PSBTs using types that aren't specified
+      in [BIP174][].  It's proposed that one type byte or a range of
+      type bytes be reserved for private PSBTs extensions, similar to
+      reserved IP address ranges for private networks.  The exact
+      construction of this mechanism was a particular focus for
+      discussion this week.
+
+    - *Global version number:* although the goal is to design enhancements
+      to PSBTs so that they're backwards compatible, Chow proposes to
+      add a version byte to PSBTs indicating the most advanced feature
+      they use so that older parsers can detect when they're being given
+      a PSBT they might not understand.  PSBTs without an explicit
+      version number would be considered to use version 0.
+
+    - *Multi-byte types:* To allow more types, multi-byte types are
+      proposed.  Mailing list discussion seems to favor using the same
+      CompactSize unsigned integers used in the Bitcoin protocol.
+
+- **Trampoline payments:** Bastien Teinturier opened a [PR][trampoline
+  pr] in the BOLTs repository and started a [discussion][trampoline
+  discussion] on the Lightning-Dev mailing list about adding support to
+  the protocol for the trampoline payments described in [Newsletter
+  #40][news40 trampoline payments] where the spender sends the payment
+  to an intermediate node who calculates the path either to another
+  intermediate node (for privacy) or to the receiving node.  This can be
+  very beneficial to low-bandwidth LN clients (such as on mobile
+  devices) that don't want to keep up with gossip traffic and so only
+  know how to route to a small number of nodes.  Sending a trampoline
+  payment will require coordination between multiple nodes and so
+  requires modification of the Lightning specification.
+
+## Bech32 sending support
+
+*Week 21 of 24 in a [series][bech32 series] about allowing the people
```

Comment:
Perhaps a more general "enabling users to access..." esp. if the audience includes wallet providers.

Reply:
That would be more general but it'd also be a bit less clear that this section is meant to be focused on getting wallets to do that bare minimum of work to accept bech32 addresses and generate native segwit scriptPubKeys.  "Enabling users to access all of segwit's benefits" sounds like we're advocating full segwit support, which (of course) we do, but at a lower priority than trying to get near-universal support for the ability to pay bech32 addresses.  (We used to have a longer introduction that spelled all that out, but at least one reader mistook its length for us repeating the whole section each week.)